indexing-service/src/main/java/org/apache/druid/indexing/common/task/AbstractBatchIndexTask.java

```diff
@@ -409,13 +413,19 @@ protected boolean tryTimeChunkLock(TaskActionClient client, List<Interval> inter
       }
     }
 
-    for (Interval interval : uniqueIntervals) {
-      final TaskLock lock = client.submit(new TimeChunkLockTryAcquireAction(TaskLockType.EXCLUSIVE, interval));
-      if (lock == null) {
-        return false;
-      }
-    }
-    return true;
+    // This method is called isReady() which is called in the Overlord to check if the task is ready for execution.
```

Comment:
This method is called by the `isReady` method elsewhere (comment: it'd be nice to explain where the `isReady` method lives...)

Reply:
Thanks. Improved the comment a bit.